modules/kmm-day1-machineconfigpool.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-day1-machineconfigpool_{context}"]
+= The MachineConfigPool
+
+The `MachineConfigPool` identifies a collection of nodes that are affected by the applied MCO.
+
+[source,yaml]
+----
+kind: MachineConfigPool
+metadata:
+  name: sfc
+spec:
+  machineConfigSelector: <1>
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker, sfc]}
+  nodeSelector: <2>
+    matchLabels:
+      node-role.kubernetes.io/sfc: ""
+  paused: false
+  maxUnavailable: 1
+----
+<1> Matches the labels in the MachineConfig.
+<2> Matches the labels on the node.
+
+There are predefined `MachineConfigPools` in the OCP cluster:
+
+* `worker`: Targets all worker nodes in the cluster
+
+* `master`: Targets all master nodes in the cluster
+
+Defining the following `MachineConfig` targets the master `MachineConfigPool`:
```

Comment:
I recommend using one gerund instead of two.

"Define the following"

Reply:
Changed to "Define the following `MachineConfig` to target the master `MachineConfigPool`:"